src/content/docs/reference/configuration.mdx

```diff
@@ -850,50 +865,239 @@ Enables parsing of [CSS modules](https://github.com/css-modules/css-modules)
 
 ### `css.formatter.enabled`
 
-Enables Biome's formatter for CSS (and its super languages) files.
+Enables Biome's formatter for CSS files.
 
 > Default: `false`
 
+```json title="biome.json"
+{
+  "css": {
+    "formatter": {
+      "enabled": false
+    }
+  }
+}
+```
+
 ### `css.formatter.indentStyle`
 
 
-The style of the indentation for CSS (and its super languages) files. It can be `"tab"` or `"space"`.
+The style of the indentation for CSS files. It can be `"tab"` or `"space"`.
 
 > Default: `"tab"`
 
 
 ### `css.formatter.indentWidth`
 
-How big the indentation should be for CSS (and its super languages) files.
+How big the indentation should be for CSS files.
 
 > Default: `2`
 
+
+```json title="biome.json"
+{
+  "css": {
+    "formatter": {
+      "indentWidth": 2
+    }
+  }
+}
+```
+
 ### `css.formatter.lineEnding`
 
-The type of line ending for CSS (and its super languages) files.
+The type of line ending for CSS  files.
 - `"lf"`, Line Feed only (`\n`), common on Linux and macOS as well as inside git repos;
 - `"crlf"`, Carriage Return + Line Feed characters (`\r\n`), common on Windows;
 - `"cr"`, Carriage Return character only (`\r`), used very rarely.
 
 > Default: `"lf"`
 
+
+
 ### `css.formatter.lineWidth`
 
-How many characters can be written on a single line in JSON (and its super languages) files.
+How many characters can be written on a single line in CSS files.
 
 > Default: `80`
 
 ### `css.formatter.quoteStyle`
+
 The type of quote used when representing string literals. It can be `"single"` or `"double"`.
 
 > Default: `"double"`
 
 ### `css.linter.enabled`
 
-Enables Biome's linter for CSS (and its super languages) files.
+Enables Biome's linter for CSS files.
+
+> Default: `true`
+
+```json title="biome.json"
+{
+  "css": {
+    "linter": {
+      "enabled": false
+    }
+  }
+}
+```
+
+### `css.assist.enabled`
+
+Enables Biome's assist for CSS files.
+
+> Default: `true`
+
+```json title="biome.json"
+{
+  "css": {
+    "assist": {
+      "enabled": false
+    }
+  }
+}
+```
+
+## `graphql`
+
+Options applied to the GraphQL files.
+
+
+### `graphql.formatter.enabled`
+
+Enables Biome's formatter for GraphQL files.
 
 > Default: `false`
 
+### `graphql.formatter.indentStyle`
+
+
+The style of the indentation for GraphQL files. It can be `"tab"` or `"space"`.
+
+> Default: `"tab"`
+
+
+### `graphql.formatter.indentWidth`
+
+How big the indentation should be for GraphQL files.
+
+> Default: `2`
+
+### `graphql.formatter.lineEnding`
+
+The type of line ending for GraphQL files.
+- `"lf"`, Line Feed only (`\n`), common on Linux and macOS as well as inside git repos;
+- `"crlf"`, Carriage Return + Line Feed characters (`\r\n`), common on Windows;
+- `"cr"`, Carriage Return character only (`\r`), used very rarely.
+
+> Default: `"lf"`
+
+### `graphql.formatter.lineWidth`
+
+How many characters can be written on a single line in √ files.
```

Comment:
```suggestion
The amount of characters that can be written on a single line in GraphQL files.
```

Reply:
This is a copy paste of all other descriptions. I will update all of them